### PR TITLE
Graphclust wrappers

### PR DESCRIPTION
**Pull request by mmiladi**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by mmiladi** [comment]
@bgruening  A draft for graphclust-wrapper conda recipes is added here. Test commands are not complete but a good start. First step is to install perl dependencies. (perl-math-round,perl-array-utils). Can you please have a look at yml and build files please? Thanks

File: recipes/graphclust-wrappers/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+#Copy binery file since there is no INSTALL part in the makefile
+cp -R galaxy_wrappers/ $PREFIX/bin
```
Comment (bgruening) [inline recipes/graphclust-wrappers/build.sh]: You probably want to have all perl scripts directly in $PREFIX/bin aren't you?

File: recipes/graphclust-wrappers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  # git_url: https://github.com/BackofenLab/GraphClust.git
+  # git_rev: 0.7.6
+
+build:
+  number: 0
+  skip: True  # [not linux64 or osx]
+
+requirements:
+  build:
+    - perl-math-round
```
Comment (bgruening) [inline recipes/graphclust-wrappers/meta.yaml:19]: dependency on `perl-threaded` is missing, also for the runtime.

File: recipes/graphclust-wrappers/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    - perl-array-utils
+
+  run:
+    - perl-math-round
+    - perl-array-utils
+
+test:
+  commands:
+    ls; ./LocARNAGraphClust/mloc2stockholm.pl -h 2>&1 | grep "USAGE"
```
Comment (bgruening) [inline recipes/graphclust-wrappers/meta.yaml]: This needs to be a list in yaml.
This should work ideally ... (only if your script are directly in your PREFIX/bin)
```
- mloc2stockholm.pl -h 2>&1 | grep "USAGE" 
```

**Pull request merged**